src/redux/slices/project-slice.ts

```diff
@@ -16,16 +18,42 @@ export const getProject = createAsyncThunk("project/getProject", async (id: numb
   }
 })
 
+export const createProject = createAsyncThunk(
+  "project/createProject",
+  async (data: ProjectFormData, thunkApi) => {
+    try {
+      const response = await axiosInstance.post("/admin/projects", data)
+      return response.data
+    } catch (error) {
+      const axiosError = error as AxiosError
+      const response = axiosError.response?.data as ApiError
+      return thunkApi.rejectWithValue(response.message)
+    }
+  }
+)
+
 interface InitialState {
   loading: Loading.Idle | Loading.Pending | Loading.Fulfilled | Loading.Rejected
   error: string | null
   project: Project | null
+  skills: Skill[]
```

Comment:
No need for this. There's already in skills-slice.ts

Reply:
Done ^^ @Dev-Nino